src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.sql

```diff
@@ -679,48 +681,52 @@ GO
 **************************************************************/
 --
 -- STORED PROCEDURE
---     AddInstance
+--      AddInstance
 --
 -- DESCRIPTION
---     Adds a DICOM instance.
+--      Adds a DICOM instance.
 --
 -- PARAMETERS
---     @studyInstanceUid
---         * The study instance UID.
---     @seriesInstanceUid
---         * The series instance UID.
---     @sopInstanceUid
---         * The SOP instance UID.
---     @patientId
---         * The Id of the patient.
---     @patientName
---         * The name of the patient.
---     @referringPhysicianName
---         * The referring physician name.
---     @studyDate
---         * The study date.
---     @studyDescription
---         * The study description.
---     @accessionNumber
---         * The accession number associated for the study.
---     @modality
---         * The modality associated for the series.
---     @performedProcedureStepStartDate
---         * The date when the procedure for the series was performed.
---     @stringExtendedQueryTags
---         * String extended query tag data
---     @longExtendedQueryTags
---         * Long extended query tag data
---     @doubleExtendedQueryTags
---         * Double extended query tag data
---     @dateTimeExtendedQueryTags
---         * DateTime extended query tag data
---     @personNameExtendedQueryTags
---         * PersonName extended query tag data
+--      @studyInstanceUid
+--          * The study instance UID.
+--      @seriesInstanceUid
+--          * The series instance UID.
+--      @sopInstanceUid
+--          * The SOP instance UID.
+--      @patientId
+--          * The Id of the patient.
+--      @patientName
+--          * The name of the patient.
+--      @referringPhysicianName
+--          * The referring physician name.
+--      @studyDate
+--          * The study date.
+--      @studyDescription
+--          * The study description.
+--      @accessionNumber
+--          * The accession number associated for the study.
+--      @modality
+--          * The modality associated for the series.
+--      @performedProcedureStepStartDate
+--          * The date when the procedure for the series was performed.
+--      @stringExtendedQueryTags
+--          * String extended query tag data
+--      @longExtendedQueryTags
+--          * Long extended query tag data
+--      @doubleExtendedQueryTags
+--          * Double extended query tag data
+--      @dateTimeExtendedQueryTags
+--          * DateTime extended query tag data
+--      @personNameExtendedQueryTags
+--          * PersonName extended query tag data
+--      @initialStatus
+--          * Initial status 
+--      @extendedQueryTagsVersion
+--          * ExtendedQueryTags Version
 -- RETURN VALUE
 --     The watermark (version).
 ------------------------------------------------------------------------
-CREATE OR ALTER PROCEDURE dbo.AddInstance
+CREATE PROCEDURE dbo.AddInstance
```

Comment:
we should create or alter, right? In case only some of this script has executed?

Reply:
Here and below

Reply:
The script is executed in a transaction on an empty Database, is it possible that some of the script is executed while others not?